frame/balances/src/lib.rs

```diff
@@ -492,6 +506,17 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::hooks]
+	impl<T: Config<I>, I: 'static> Hooks<T::BlockNumber> for Pallet<T, I> {
+		#[cfg(not(feature = "zero_ed"))]
+		fn integrity_test() {
+			frame_support::defensive_assert!(
```

Comment:
```suggestion
			assert!(
```
This hook is std-only.